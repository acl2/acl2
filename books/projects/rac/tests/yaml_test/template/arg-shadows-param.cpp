// RAC begin

template <int i>
int f(int i) {
  return 0;
}
