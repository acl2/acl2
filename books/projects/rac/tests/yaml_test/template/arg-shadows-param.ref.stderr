arg-shadows-param.cpp:4 (10-11): Duplicate identifier declaration (could be an enum, a variable template parameter or function) `i`
  4 | int f(int i) {
    |           ^
note: first defined here
  3 | template <int i>
    |           ^^^^^
